apps/web/app/future/settings/organizations/[id]/about/page.tsx

```diff
@@ -1,11 +1,15 @@
-import LegacyPage, { WrappedAboutOrganizationPage } from "@pages/settings/organizations/[id]/about";
 import { _generateMetadata } from "app/_utils";
 import { WithLayout } from "app/layoutHOC";
 
+import LegacyPage, { LayoutWrapper } from "~/settings/organizations/[id]/about-view";
+
 export const generateMetadata = async () =>
   await _generateMetadata(
     (t) => t("about_your_organization"),
     (t) => t("about_your_organization_description")
   );
 
-export default WithLayout({ Page: LegacyPage, getLayout: WrappedAboutOrganizationPage });
+export default WithLayout({
```

Comment:
In #16312 I relied on a single `layout.tsx` file in the root of the settings routes. It feels like `WithLayout` is nice when we're migrating `getServerSideProps` to app routers but in this instance we can reduce the boilerplate in each file.

It would also be easier to see where the layout is located. I know there was a case where someone was confused why the layout was duplicating on a page.

Reply:
I agree with you. But I suggest that we do this after (1) we finish adding remaining routes to App Router and (2) we finish extracting pages in `/pages` to `/modules`.